tests/README.md

```diff
@@ -0,0 +1,7 @@
+# FreshRSS tests
+
+```sh
+cd ./tests/
+wget https://phar.phpunit.de/phpunit.phar
```

Comment:
@aledeg FYI, tests for UserQueryTest are broken due to https://github.com/sebastianbergmann/phpunit/wiki/Release-Announcement-for-PHPUnit-4.0.0#backwards-compatibility-issues